1.1/spec/18-Annex-C.adoc

```diff
@@ -1,6 +1,149 @@
 === Annex C 
 === (informative)
 
+=== Best-practice: A GeoSPARQL-enabled triple store as a backend of an OGC web service
+
+This annex presents best practices for the creation of Linked Data backends to OGC webservices.
+
+==== C.1 Accessing geospatial features in a SPARQL endpoint
+
+Geospatial _features_ accessed via SPARQL endpoints <<SPARQLPROT>> are, as defined in the GeoSPARQL standard, instances of http://www.opengis.net/ont/geosparql#Feature[geo:Feature] or of subclasses of it.
+They may include one or more http://www.opengis.net/ont/geosparql#hasGeometry[geo:hasGeometry] relations and further properties related to the feature and may be grouped into http://www.opengis.net/ont/geosparql#FeatureCollection[geo:FeatureCollection] instances where `geo:FeatureCollection` is a new class in GeoSPARQL 1.1 specifically for the description of collections of `geo:Feature` instances.
+
+The following example SPARQL query retrieves all feature collections of a given SPARQL endpoint. A third-party webservice may convert the query results to one or many geoformat serializations.  
```

Comment:
feature collections → Feature Collections (class names are capitalised).

Reply:
I would prefer just "data formats" or something like that instead of "geoformat serializations"